lms/static/sass/shared/_footer-edx.scss

```diff
@@ -7,17 +7,67 @@
 $edx-footer-link-color: $link-color;
 $edx-footer-bg-color: rgb(252,252,252);
 
-
 // Aggressively scoped for Drupal
 // ==============================
 // These styles are being loaded on Drupal, LMS and WordPress
 // sites so the scope has to be aggressive to override default
 // CMS styles
 footer#footer-edx-v3 {
   background: $edx-footer-bg-color;
-  padding: 20px;
+  padding: 15px 0;
   border-top: 1px solid $courseware-button-border-color;
 
+  .container-bootstrap {
+    padding-right: 15px;
+    padding-left: 15px;
```

Comment:
Use https://github.com/edx/edx-platform/blob/master/lms/static/sass/vendor/bi-app/_mixins.scss#L39-L52 for padding right and left